templates/perfalert_resolved_regression.html

```diff
@@ -0,0 +1,39 @@
+<p>The following bug list contains performance alerts whose resolution has changed in the last week:</p>
+<table {{ table_attrs }}>
+    <thead>
+        <tr>
+            <th>Bug</th>
+            <th>Summary</th>
+            <th>Resolution</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for i, (bugid, summary) in enumerate(data) -%}
+            <tr {% if i % 2 == 0 %}bgcolor="#E0E0E0"
+            {% endif -%}
+            >
+            <td>
+                <a href="https://bugzilla.mozilla.org/show_bug.cgi?id={{ bugid }}">{{ bugid }}</a>
+            </td>
+            <td>
+                <table>
```

Comment:
Just a note that I didn't add the table_attrs here because with those settings, the inner table shows up with black borders around each section.